### PR TITLE
#140 - Refine the Admin interface.

### PR DESCRIPTION
**Pull request by kvig**
This PR cleans up the Admin interface.

It removes most of the entries in the Admin Interface, the header is made more consistent with the application and
Users/Teams/Environments are able to be seen from each other. Users may be assigned roles directly from the Team, Environment and User pages.

### CODE REVIEW

File: functionary/core/admin/team.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        # what we want.
+        if db_field.attname == "user_id":
+            field.widget = field.widget.widget
+        return field
+
+
+class TeamForm(forms.ModelForm):
+    """Custom form to show the environments associated with the team"""
+
+    environments = EnvironmentChoiceField(
```
Comment (scott-taubman) [inline functionary/core/admin/team.py:37]: This ends up getting displayed as a multi-select box where your selection doesn't do anything.  Something like the list that you have for users, except it's read-only and just links over to the environments might be more appropriate.

File: functionary/core/admin/team.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    class Meta:
+        model = Team
+        fields = ["name"]
+
+    def __init__(self, *args, **kwargs):
+        super(TeamForm, self).__init__(*args, **kwargs)
+        # Since the Team doesn't have a direct reference to the
+        # Environments, add the queryset manually for this Team
+        if self.instance and "environments" in self.fields:
+            self.fields["environments"].queryset = Environment.objects.filter(
```
Comment (scott-taubman) [inline functionary/core/admin/team.py:51]: You can get to the environments through the Team instance:

```python
self.instance.environments
```

File: functionary/ui/templates/admin/base_site.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                        </div>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock branding %}
+{% block usertools %}
+{% endblock usertools %}
+{% block nav-sidebar %}
```
Comment (scott-taubman) [inline functionary/ui/templates/admin/base_site.html:73]: There's a fly-out panel on the left side of the list and detail pages for models that broke somehow.  A javascript error shows up in the console and when you click the >> on the collapsed panel nothing happens.  I don't know what's causing that, but I'm guessing it's something in these template customizations.

**Review by scott-taubman — COMMENTED** [review]
Made some comments about things I think could be cleaner, but for the most part looks good.

File: functionary/core/admin/team.py (hunk 1 above)
Comment (kvig) [reply functionary/core/admin/team.py:40]: Changed the list to readonly. I didn't see a widget that I could change it to in order to just display links. For a field on a model, that's handled by rendering a div on a template. This field is only defined on the form and makes things more challenging.

**Review by scott-taubman — COMMENTED** [review]
I think this does the job for now.  We can further refine things as we use it and discover the pain points.

**Pull request merged**